packages/jest-schemas/src/raw-types.ts

```diff
@@ -0,0 +1,351 @@
+  Type.Literal('html'),
+  Type.Literal('json'),
+  Type.Literal('json-summary'),
+  Type.Literal('lcov'),
+  Type.Literal('lcovonly'),
+  Type.Literal('none'),
+  Type.Literal('teamcity'),
+  Type.Literal('text'),
+  Type.Literal('text-lcov'),
+  Type.Literal('text-summary'),
+]);
+
+const RawCoverageReporters = Type.Array(
+  Type.Union([
+    RawCoverageReporterNames,
+    Type.Tuple([
+      RawCoverageReporterNames,
+      Type.Record(Type.String(), Type.Unknown()),
+    ]),
+  ]),
+);
+
+const RawGlobalFakeTimersConfig = Type.Partial(
+  Type.Object({
+    enableGlobally: Type.Boolean({
+      description:
+        'Whether fake timers should be enabled globally for all test files.',
+      default: false,
+    }),
+  }),
+);
+
+const RawFakeableAPI = Type.Union([
+  Type.Literal('Date'),
+  Type.Literal('hrtime'),
+  Type.Literal('nextTick'),
+  Type.Literal('performance'),
+  Type.Literal('queueMicrotask'),
+  Type.Literal('requestAnimationFrame'),
+  Type.Literal('cancelAnimationFrame'),
+  Type.Literal('requestIdleCallback'),
+  Type.Literal('cancelIdleCallback'),
+  Type.Literal('setImmediate'),
+  Type.Literal('clearImmediate'),
+  Type.Literal('setInterval'),
+  Type.Literal('clearInterval'),
+  Type.Literal('setTimeout'),
+  Type.Literal('clearTimeout'),
+]);
+
+const RawFakeTimersConfig = Type.Partial(
+  Type.Object({
+    advanceTimers: Type.Union([Type.Boolean(), Type.Number({minimum: 0})], {
+      description:
+        'If set to `true` all timers will be advanced automatically by 20 milliseconds every 20 milliseconds. A custom ' +
+        'time delta may be provided by passing a number.',
+      default: false,
+    }),
+    doNotFake: Type.Array(RawFakeableAPI, {
+      description:
+        'List of names of APIs (e.g. `Date`, `nextTick()`, `setImmediate()`, `setTimeout()`) that should not be faked.' +
+        '\n\nThe default is `[]`, meaning all APIs are faked.',
+      default: [],
+    }),
+    now: Type.Union([Type.Integer({minimum: 0}), Type.Date()], {
+      description:
+        'Sets current system time to be used by fake timers.\n\nThe default is `Date.now()`.',
+    }),
+    timerLimit: Type.Number({
+      description:
+        'The maximum number of recursive timers that will be run when calling `jest.runAllTimers()`.',
+      default: 100_000,
+      minimum: 0,
+    }),
+    legacyFakeTimers: Type.Literal(false, {
+      description:
+        'Use the old fake timers implementation instead of one backed by `@sinonjs/fake-timers`.',
+      default: false,
+    }),
+  }),
+);
+
+const RawLegacyFakeTimersConfig = Type.Partial(
+  Type.Object({
+    legacyFakeTimers: Type.Literal(true, {
+      description:
+        'Use the old fake timers implementation instead of one backed by `@sinonjs/fake-timers`.',
+      default: true,
+    }),
+  }),
+);
+
+export const RawFakeTimers = Type.Intersect([
+  RawGlobalFakeTimersConfig,
+  Type.Union([RawFakeTimersConfig, RawLegacyFakeTimersConfig]),
+]);
```

Comment:
this is currently

```ts
type CoverageThreshold = {
  [path: string]: CoverageThresholdValue;
  global: CoverageThresholdValue;
};
```

I don't think my current approach is the correct "translation" of that, but it seems to work in practice